examples/ann/annoy_nsmlib_example.ipynb

```diff
@@ -0,0 +1,711 @@
+    "\n",
+    "Firstly, let's import the data and build the Light FM model, as per the example in th docs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from sklearn.metrics import pairwise_distances\n",
+    "import timeit\n",
+    "import time\n",
+    "\n",
+    "from lightfm.datasets import fetch_movielens\n",
+    "\n",
+    "movielens = fetch_movielens()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "('test', <class 'scipy.sparse.coo.coo_matrix'>, (943, 1682))\n",
+      "('item_features', <class 'scipy.sparse.csr.csr_matrix'>, (1682, 1682))\n",
+      "('train', <class 'scipy.sparse.coo.coo_matrix'>, (943, 1682))\n",
+      "('item_labels', <type 'numpy.ndarray'>, (1682,))\n",
+      "('item_feature_labels', <type 'numpy.ndarray'>, (1682,))\n"
+     ]
+    }
+   ],
+   "source": [
+    "for key, value in movielens.items():\n",
+    "    print(key, type(value), value.shape)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "train = movielens['train']\n",
+    "test = movielens['test']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Precision: train 0.74, test 0.09.\n",
+      "AUC: train 0.97, test 0.91.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from lightfm import LightFM\n",
+    "from lightfm.evaluation import precision_at_k\n",
+    "from lightfm.evaluation import auc_score\n",
+    "\n",
+    "model = LightFM(learning_rate=0.05, loss='warp', no_components=64, item_alpha=0.001)\n",
+    "\n",
+    "model.fit_partial(train, item_features=movielens['item_features'], epochs=20 )\n",
+    "\n",
+    "train_precision = precision_at_k(model, train, k=10).mean()\n",
+    "test_precision = precision_at_k(model, test, k=10).mean()\n",
+    "\n",
+    "train_auc = auc_score(model, train).mean()\n",
+    "test_auc = auc_score(model, test).mean()\n",
+    "\n",
+    "print('Precision: train %.2f, test %.2f.' % (train_precision, test_precision))\n",
+    "print('AUC: train %.2f, test %.2f.' % (train_auc, test_auc))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can get the item vectors by simply multiplying the item features by the models item feature embeddings:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "item_vectors = movielens['item_features'] * model.get_item_representations()[1]"
```

Comment:
Could you call `model.get_item_representations(movielens['item_features'])` instead of the multiplication (as per the [docs](http://lyst.github.io/lightfm/docs/lightfm.html#lightfm.LightFM.get_item_representations)). The effect will be the same, but it's probably a more straightforward experience for the user.